Pipfile

```diff
@@ -34,6 +34,7 @@ django-cors-headers = "==2.4.0"
 django-redis = "*"
 celery = {extras = ["redis"],version = "==4.2.1"}
 seed-isort-config = "==1.6.0"
+pytest-django = "*"
```

Comment:
@rnegron any chance we can set this up as a dev dependency?

Reply:
I wanted to, but then realized it would not be possible to run it with `docker-compose exec` unless the image installs the Pipfile dev dependencies... What do you think?

Reply:
IMO one way to do this is to add a `test` service to the `docker-compose.yml` that executes a shell script with what you need to do for testing. This could also be used in our CircleCI config.

Reply:
Well, that would work but running `docker-compose up` will also start the test service, which may or may not be what you want at the time. Currently there is no good way to conditionally enable services in Docker. That said, I will commit that suggestion as it does solve the dev dependency issue.